OneSignalSDK/onesignal/src/main/java/com/onesignal/OSInAppMessageController.java

```diff
@@ -448,24 +435,21 @@ void onFailure(int statusCode, String response, Throwable throwable) {
      * For click counting, every message has it click id array
      * */
     private void setDataForRedisplay(OSInAppMessage message) {
-        if (!message.getDisplayStats().isRedisplayEnabled())
```

Comment:
why is this deleted?

Reply:
This is why im always trying to rename things, we want to have redisplay data for IAMs so when we save them to SQL it matches the scheme and nothing is NULL

Redisplay isnt something we "enable" and "disable" also its just whether or not the IAM will show more than once. I think every IAM should have the same scheme and should be cached together, not separately